byway/src/main/webapp/routepage.html

```diff
@@ -34,6 +34,12 @@
       <p id="general-message">Showing limited results due to: </p>
       <p id="statuses"></p>
     </div>
+
+    <div class = "tips">
```

Comment:
maybe add something about how to remove stops in the tips

Reply:
Good point, added!